app/packages/looker-3d/src/action-bar/ViewHelp.tsx

```diff
@@ -4,15 +4,19 @@ import { ActionItem } from "../containers";
 import { ACTION_VIEW_HELP } from "../constants";
 
 export const LOOKER3D_HELP_ITEMS = [
-  { shortcut: "Left mouse drag", title: "Rotate", detail: "Rotate the camera" },
-  { shortcut: "Wheel", title: "Zoom", detail: "Zoom in and out" },
+  { shortcut: "Left drag", title: "Rotate", detail: "Rotate the camera" },
+  { shortcut: "T", title: "Top-down", detail: "Reset camera to top-down view" },
   {
-    shortcut: "Middle mouse drag",
-    title: "Translate",
-    detail: "Translate the camera",
+    shortcut: "Ctrl + left drag / Wheel",
```

Comment:
Perhaps I don't fully appreciate the left/right clicks as a single button mouse user, but would this more concise set of instructions suffice?

```
Wheel           Zoom        Zoom in and out
Shift + drag    Translate   Translate the camera
```